hackathon_site/dashboard/frontend/src/slices/event/teamDetailSlice.ts

```diff
@@ -0,0 +1,99 @@
+import {
+    createAsyncThunk,
+    createEntityAdapter,
+    createSelector,
+    createSlice,
+} from "@reduxjs/toolkit";
+import { AppDispatch, RootState } from "slices/store";
+import { displaySnackbar } from "slices/ui/uiSlice";
+import { get } from "api/api";
+import { ProfileWithUser, Team } from "api/types";
+
+const extraState = {
```

Comment:
You need an interface to type your extra state

Reply:
addeddd :)